From 7d9b6b9d40799bc97aff03085141fedfe841cc4d Mon Sep 17 00:00:00 2001
From: David Benjamin <davidben@google.com>
Date: Thu, 16 Feb 2023 17:37:57 -0500
Subject: [PATCH] Revert go:build ignore lines

This convention seems to break with some other tooling we have. Until we
figure out how to resolve that, remove the lines.

This partially reverts 54b04fdc21d540a6e24f9ddb7ddc3e583518e24f but
keeps the fixes to the license header comments.

Change-Id: I4f08a9f3daf65d17b4c78ac6f4ac3de234ec3436
Reviewed-on: https://boringssl-review.googlesource.com/c/boringssl/+/57366
Reviewed-by: Bob Beck <bbe@google.com>
Commit-Queue: Bob Beck <bbe@google.com>
Commit-Queue: David Benjamin <davidben@google.com>
Auto-Submit: David Benjamin <davidben@google.com>
(cherry picked from commit bab2f96e2637acb107fe9b099c58befbca918748)
---
 crypto/err/err_data_generate.go                        | 2 --
 crypto/fipsmodule/bn/bn_test_to_fuzzer.go              | 2 --
 crypto/fipsmodule/bn/check_bn_tests.go                 | 2 --
 crypto/fipsmodule/ec/make_ec_scalar_base_mult_tests.go | 2 --
 crypto/fipsmodule/ec/make_p256-nistz-tests.go          | 2 --
 crypto/fipsmodule/ec/make_tables.go                    | 2 --
 crypto/obj/objects.go                                  | 2 --
 crypto/x509/test/make_basic_constraints.go             | 2 --
 crypto/x509/test/make_invalid_extensions.go            | 2 --
 crypto/x509/test/make_many_constraints.go              | 2 --
 crypto/x509/test/make_policy_certs.go                  | 2 --
 util/all_tests.go                                      | 2 --
 util/check_filenames.go                                | 2 --
 util/check_imported_libraries.go                       | 2 --
 util/check_stack.go                                    | 2 --
 util/compare_benchmarks.go                             | 2 --
 util/convert_comments.go                               | 2 --
 util/convert_wycheproof.go                             | 2 --
 util/diff_asm.go                                       | 2 --
 util/doc.go                                            | 2 --
 util/embed_test_data.go                                | 2 --
 util/fetch_ech_config_list.go                          | 2 --
 util/fipstools/acvp/acvptool/test/check_expected.go    | 2 --
 util/fipstools/acvp/acvptool/test/trim_vectors.go      | 2 --
 util/fipstools/break-hash.go                           | 2 --
 util/fipstools/break-kat.go                            | 2 --
 util/godeps.go                                         | 2 --
 util/make_errors.go                                    | 2 --
 util/make_prefix_headers.go                            | 2 --
 util/read_symbols.go                                   | 2 --
 util/run_android_tests.go                              | 2 --
 31 files changed, 62 deletions(-)

diff --git a/crypto/err/err_data_generate.go b/crypto/err/err_data_generate.go
index 0c57f3512..f7bf75a45 100644
--- a/crypto/err/err_data_generate.go
+++ b/crypto/err/err_data_generate.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 package main
 
 import (
diff --git a/crypto/fipsmodule/bn/bn_test_to_fuzzer.go b/crypto/fipsmodule/bn/bn_test_to_fuzzer.go
index 2915db5f9..1e48df350 100644
--- a/crypto/fipsmodule/bn/bn_test_to_fuzzer.go
+++ b/crypto/fipsmodule/bn/bn_test_to_fuzzer.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 package main
 
 import (
diff --git a/crypto/fipsmodule/bn/check_bn_tests.go b/crypto/fipsmodule/bn/check_bn_tests.go
index 032b9e328..dd1def80d 100644
--- a/crypto/fipsmodule/bn/check_bn_tests.go
+++ b/crypto/fipsmodule/bn/check_bn_tests.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 package main
 
 import (
diff --git a/crypto/fipsmodule/ec/make_ec_scalar_base_mult_tests.go b/crypto/fipsmodule/ec/make_ec_scalar_base_mult_tests.go
index d97ca5650..6aea66415 100644
--- a/crypto/fipsmodule/ec/make_ec_scalar_base_mult_tests.go
+++ b/crypto/fipsmodule/ec/make_ec_scalar_base_mult_tests.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 package main
 
 import (
diff --git a/crypto/fipsmodule/ec/make_p256-nistz-tests.go b/crypto/fipsmodule/ec/make_p256-nistz-tests.go
index e10990b30..04a1a1c87 100644
--- a/crypto/fipsmodule/ec/make_p256-nistz-tests.go
+++ b/crypto/fipsmodule/ec/make_p256-nistz-tests.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 package main
 
 import (
diff --git a/crypto/fipsmodule/ec/make_tables.go b/crypto/fipsmodule/ec/make_tables.go
index d4e48cd3b..23b87ce73 100644
--- a/crypto/fipsmodule/ec/make_tables.go
+++ b/crypto/fipsmodule/ec/make_tables.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 package main
 
 import (
diff --git a/crypto/obj/objects.go b/crypto/obj/objects.go
index 719061eb0..ae1346af2 100644
--- a/crypto/obj/objects.go
+++ b/crypto/obj/objects.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 package main
 
 import (
diff --git a/crypto/x509/test/make_basic_constraints.go b/crypto/x509/test/make_basic_constraints.go
index ea502b4c8..67f4b913d 100644
--- a/crypto/x509/test/make_basic_constraints.go
+++ b/crypto/x509/test/make_basic_constraints.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 // make_basic_constraints.go generates self-signed certificates with the basic
 // constraints extension.
 package main
diff --git a/crypto/x509/test/make_invalid_extensions.go b/crypto/x509/test/make_invalid_extensions.go
index 8287bf8d3..884d93355 100644
--- a/crypto/x509/test/make_invalid_extensions.go
+++ b/crypto/x509/test/make_invalid_extensions.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 // make_invalid_extensions.go generates a number of certificate chains with
 // invalid extension encodings.
 package main
diff --git a/crypto/x509/test/make_many_constraints.go b/crypto/x509/test/make_many_constraints.go
index 24a5c4071..07f2edc76 100644
--- a/crypto/x509/test/make_many_constraints.go
+++ b/crypto/x509/test/make_many_constraints.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 // make_many_constraints.go generates test certificates many_constraints.pem,
 // many_names*.pem, and some_names*.pem for x509_test.cc
 package main
diff --git a/crypto/x509/test/make_policy_certs.go b/crypto/x509/test/make_policy_certs.go
index 7ba087df1..cf09ee8e4 100644
--- a/crypto/x509/test/make_policy_certs.go
+++ b/crypto/x509/test/make_policy_certs.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 // make_policy_certs.go generates certificates for testing policy handling.
 package main
 
diff --git a/util/all_tests.go b/util/all_tests.go
index fb8b857c5..146b98be6 100644
--- a/util/all_tests.go
+++ b/util/all_tests.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 package main
 
 import (
diff --git a/util/check_filenames.go b/util/check_filenames.go
index 384c19f51..886c3f609 100644
--- a/util/check_filenames.go
+++ b/util/check_filenames.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 // check_filenames.go checks that filenames are unique. Some of our consumers do
 // not support multiple files with the same name in the same build target, even
 // if they are in different directories.
diff --git a/util/check_imported_libraries.go b/util/check_imported_libraries.go
index f3803f1c1..187e51441 100644
--- a/util/check_imported_libraries.go
+++ b/util/check_imported_libraries.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 // check_imported_libraries.go checks that each of its arguments only imports
 // allowed libraries. This is used to avoid accidental dependencies on
 // libstdc++.so.
diff --git a/util/check_stack.go b/util/check_stack.go
index ad763e652..b718ea4ac 100644
--- a/util/check_stack.go
+++ b/util/check_stack.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 // check_stack.go checks that each of its arguments has a non-executable stack.
 // See https://www.airs.com/blog/archives/518 for details.
 package main
diff --git a/util/compare_benchmarks.go b/util/compare_benchmarks.go
index 05e1b5db0..d631f832b 100644
--- a/util/compare_benchmarks.go
+++ b/util/compare_benchmarks.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 // compare_benchmarks takes the JSON-formatted output of bssl speed and
 // compares it against a baseline output.
 package main
diff --git a/util/convert_comments.go b/util/convert_comments.go
index df9e3d3ae..917f29c86 100644
--- a/util/convert_comments.go
+++ b/util/convert_comments.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 package main
 
 import (
diff --git a/util/convert_wycheproof.go b/util/convert_wycheproof.go
index f7997a4aa..76aff175f 100644
--- a/util/convert_wycheproof.go
+++ b/util/convert_wycheproof.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 // convert_wycheproof.go converts Wycheproof test vectors into a format more
 // easily consumed by BoringSSL.
 package main
diff --git a/util/diff_asm.go b/util/diff_asm.go
index 5ac1c04bb..27c0aa20a 100644
--- a/util/diff_asm.go
+++ b/util/diff_asm.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 package main
 
 import (
diff --git a/util/doc.go b/util/doc.go
index a76a4d091..443016dc6 100644
--- a/util/doc.go
+++ b/util/doc.go
@@ -1,5 +1,3 @@
-//go:build ignore
-
 // doc generates HTML files from the comments in header files.
 //
 // doc expects to be given the path to a JSON file via the --config option.
diff --git a/util/embed_test_data.go b/util/embed_test_data.go
index ae7135fae..0a0d1e3f1 100644
--- a/util/embed_test_data.go
+++ b/util/embed_test_data.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 // embed_test_data generates a C++ source file which exports a function,
 // GetTestData, which looks up the specified data files.
 package main
diff --git a/util/fetch_ech_config_list.go b/util/fetch_ech_config_list.go
index 732d0d3b8..8f09e66b1 100644
--- a/util/fetch_ech_config_list.go
+++ b/util/fetch_ech_config_list.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 package main
 
 import (
diff --git a/util/fipstools/acvp/acvptool/test/check_expected.go b/util/fipstools/acvp/acvptool/test/check_expected.go
index e6fd8ea81..fc2e0c7d2 100644
--- a/util/fipstools/acvp/acvptool/test/check_expected.go
+++ b/util/fipstools/acvp/acvptool/test/check_expected.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 package main
 
 import (
diff --git a/util/fipstools/acvp/acvptool/test/trim_vectors.go b/util/fipstools/acvp/acvptool/test/trim_vectors.go
index 703f75fd2..53e970e02 100644
--- a/util/fipstools/acvp/acvptool/test/trim_vectors.go
+++ b/util/fipstools/acvp/acvptool/test/trim_vectors.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 // trimvectors takes an ACVP vector set file and discards all but a single test
 // from each test group. This hope is that this achieves good coverage without
 // having to check in megabytes worth of JSON files.
diff --git a/util/fipstools/break-hash.go b/util/fipstools/break-hash.go
index a4ab8083d..e37da7525 100644
--- a/util/fipstools/break-hash.go
+++ b/util/fipstools/break-hash.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 // break-hash parses an ELF binary containing the FIPS module and corrupts the
 // first byte of the module. This should cause the integrity check to fail.
 package main
diff --git a/util/fipstools/break-kat.go b/util/fipstools/break-kat.go
index ebf5dd703..6eace5b46 100644
--- a/util/fipstools/break-kat.go
+++ b/util/fipstools/break-kat.go
@@ -1,5 +1,3 @@
-//go:build
-
 // break-kat corrupts a known-answer-test input in a binary and writes the
 // corrupted binary to stdout. This is used to demonstrate that the KATs in the
 // binary notice the error.
diff --git a/util/godeps.go b/util/godeps.go
index 56be55944..960faa46b 100644
--- a/util/godeps.go
+++ b/util/godeps.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 // godeps prints out dependencies of a package in either CMake or Make depfile
 // format, for incremental rebuilds.
 //
diff --git a/util/make_errors.go b/util/make_errors.go
index 018845258..4e2718b8d 100644
--- a/util/make_errors.go
+++ b/util/make_errors.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 package main
 
 import (
diff --git a/util/make_prefix_headers.go b/util/make_prefix_headers.go
index 8787654b5..b39e4ec7c 100644
--- a/util/make_prefix_headers.go
+++ b/util/make_prefix_headers.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 // This program takes a file containing newline-separated symbols, and generates
 // boringssl_prefix_symbols.h, boringssl_prefix_symbols_asm.h, and
 // boringssl_prefix_symbols_nasm.inc. These header files can be used to build
diff --git a/util/read_symbols.go b/util/read_symbols.go
index 58bc16b28..b5eb025a2 100644
--- a/util/read_symbols.go
+++ b/util/read_symbols.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 // read_symbols scans one or more .a files and, for each object contained in
 // the .a files, reads the list of symbols in that object file.
 package main
diff --git a/util/run_android_tests.go b/util/run_android_tests.go
index 4acf9e8ac..057e346d9 100644
--- a/util/run_android_tests.go
+++ b/util/run_android_tests.go
@@ -12,8 +12,6 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//go:build ignore
-
 package main
 
 import (
